soh/include/z64.h

```diff
@@ -1522,7 +1522,7 @@ typedef struct {
     /* 0x34 */ s32 isEnabled;
 } StickDirectionPrompt;
 
-typedef struct {
+typedef struct FileChooseContext {
```

Comment:
```suggestion
typedef struct {
```
no need to touch this

Reply:
This is actually needed for it to compile.
Also, upstream does this with every struct, we just haven't caught up yet.